core/src/main/java/org/openedx/core/ui/theme/AppColors.kt

```diff
@@ -47,6 +47,10 @@ data class AppColors(
 
     val authFacebookButtonBackground: Color,
     val authMicrosoftButtonBackground: Color,
+
+    val unitHorizontalProgressCompleted: Color,
```

Comment:
The colors are used for components and not for units. Can we please update these here and on other places as well. 🤐

Reply:
@HamzaIsrar12 it's not clear what should be updated? do you mean change the name of fields like `horizontalProgressCompletedBackground`?

Reply:
Sorry for the confusion. In this hierarchy: 
```
Course (root)
    > Section (chapter)
        > Subsection (sequential)
            > Unit (vertical)
                > Component (xBlock e.g. html, discussion, video etc)
```

These colors are used for components and not units. So we can rename the variable to `componentHorizontalProgressCompleted`, `descendantHorizontalProgressCompleted` or maybe something else you find helpful.

And same goes for other variables in the Theme, Colors and AppColors files.

Reply:
oh, right now it's clear, thank you for the explanation!

Reply:
done